packages/data-context/src/actions/WizardActions.ts

```diff
@@ -146,23 +146,30 @@ export class WizardActions {
   async scaffoldTestingType () {
     const { currentTestingType, wizard: { chosenLanguage } } = this.ctx.coreData
 
-    assert(currentTestingType)
-    assert(chosenLanguage)
+    if (!currentTestingType || !chosenLanguage) {
+      return
+    }
 
     switch (currentTestingType) {
       case 'e2e': {
         this.ctx.coreData.scaffoldedFiles = await this.scaffoldE2E()
         this.ctx.lifecycleManager.refreshMetaState()
         this.ctx.actions.project.setForceReconfigureProjectByTestingType({ forceReconfigureProject: false, testingType: 'e2e' })
 
-        return chosenLanguage
+        break
       }
       case 'component': {
+        const { chosenBundler, chosenFramework } = this.ctx.wizard
+
+        if (!chosenBundler || !chosenFramework) {
+          return
```

Comment:
`break` or `return`? pick one to use to exit this function, you have a mix in here :smile: my vote is on `return` since it is clearer that the function is exiting

Reply:
Yeah I'll make this consistent

Reply:
@tgriesser